pkg/R/DataFrame.R

```diff
@@ -281,6 +359,34 @@ setMethod("repartition",
             dataFrame(sdf)     
           })
 
+#' toJSON
+#'
+#' Convert the rows of a DataFrame into JSON objects and return a StringRRDD
```

Comment:
StringRRDD is an internal detail - lets call this `return an RDD where each element contains a JSON string`